.github/actions/setup-dfx/action.yml

```diff
@@ -4,32 +4,8 @@ description: Setup DFX
 runs:
   using: 'composite'
   steps:
-    - name: Cache DFX
-      uses: actions/cache@v3
-      with:
-        path: /usr/local/bin/dfx
-        key: dfx-cache-${{ env.dfx_version }}
-
     - name: Install dfx
       uses: dfinity/setup-dfx@main
-      with:
-        dfx-version: '0.18.0'
```

Comment:
Without providing this parameter, the GitHub action will auto-detect the desired `dfx` version from `dfx.json`